### PR TITLE
Learning unit checkbox component

### PR DESCRIPTION
**Pull request by rtichauerv**
I created a curriculum page that only shows my component.
The component is a checkbox for each learning unit, that receives a learning_unit and a previous completion_state for this learning unit, and manages its state.

### CODE REVIEW

File: components/LearningUnitCheckbox.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    transform: "translate(-50%, -50%)",
+    width: 600,
+    bgcolor: "background.paper",
+    border: "2px solid #000",
+    boxShadow: 24,
+    p: 4,
+  };
+  return (
+    <Box sx={{ ...style, width: 300 }}>
+      <h1>here it goes</h1>
```
Comment (iaheck) [inline components/LearningUnitCheckbox.js:43]: can't read in dark mode
image

**Review by iaheck — CHANGES_REQUESTED** [review]
I would just resolve the dark mode issue, even if it's just a placeholder

**Review by jjsantos01 — APPROVED** [review]
great job here! Only a minor comment that doesn't change the functtion of the component

File: pages/curriculums/[curriculumId].js [hunk 2]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { OpenAPIProvider } from "react-openapi-client";
+import LearningUnitCheckbox from "../../components/LearningUnitCheckbox";
+
+export default function LearningUnits() {
+  const isCompleted = false;
+  return (
+    <React.StrictMode>
+      <OpenAPIProvider definition="/api-docs/v1/swagger.json">
```
[the remark this answers is not in the document]
Comment (rtichauerv) [reply pages/curriculums/[curriculumId].js:9]: 🤯

[the remark this answers is not in the document]
Comment (rtichauerv) [reply pages/curriculums/[curriculumId].js]: Done

File: components/LearningUnitCheckbox.js (hunk 1 above)
Comment (rtichauerv) [reply components/LearningUnitCheckbox.js]: Done

**Review by iaheck — APPROVED** [review]

**Pull request merged**